public/js/app/pages/MemoEditingContainer.jsx

```diff
@@ -127,12 +127,10 @@ function MemoContainer(props) {
   }
 
   const courseSubHeader = () => {
-    const { title, titleOther, credits, creditUnitAbbr } = memoData
+    const { title, credits, creditUnitAbbr } = memoData
 
     const creditsStandard = credits || ''
-    const courseTitle = `${courseCode} ${userLangIndex === memoLangIndex ? title : titleOther} ${creditsStandard} ${
-      userLangIndex === 1 ? creditUnitAbbr : 'credits'
-    }`
+    const courseTitle = `${courseCode} ${title} ${creditsStandard} ${userLangIndex === 1 ? creditUnitAbbr : 'credits'}`
```

Comment:
I'm now sure about the lang of the title here. I think we might need to add something corresponding to `titleOther` in the data from Ladok.